src/components/UnreadActionIndicator.js

```diff
@@ -5,7 +5,7 @@ import Text from './Text';
 import withLocalize, {withLocalizePropTypes} from './withLocalize';
 
 const UnreadActionIndicator = props => (
-    <View accessibilityLabel={props.translate('accessibilityHints.newMessageLineIndicator')} data-action-id={props.reportActionID} style={styles.unreadIndicatorContainer}>
+    <View accessibilityLabel={props.translate('accessibilityHints.newMessageLineIndicator')} data-action-id={props.reportActionID} style={[styles.unreadIndicatorContainer, styles.userSelectNone]}>
```

Comment:
Shoot, looks like this is slightly too long 🤷 

```suggestion
    <View
        accessibilityLabel={props.translate('accessibilityHints.newMessageLineIndicator')}
        data-action-id={props.reportActionID}
        style={[styles.unreadIndicatorContainer, styles.userSelectNone]}
    >
```